vars/starter.yml

```diff
@@ -0,0 +1,24 @@
+---
+drupal_build_composer_project: true
+drupal_composer_dependencies: []
+# Presently pointing at the main branch.
+drupal_composer_project_package: 'islandora/islandora-starter-site:dev-main'
```

Comment:
This should eventually be pointed at whatever reference structure we move to use for releases... or maybe left without a version reference to pull whatever "latest" version?